dsrs/redis/README.md

```diff
@@ -1,6 +1,6 @@
-# Data Services Reference Stack(DSRS) - Redis* 
+# Data Services Reference Stack(DSRS) - Redis 
```

Comment:
@mdreyesm Hi Miguel, probably you might want to put back `*` to reference to `
*\*Other names and brands may be claimed as the property of others*` legend for Redis.

This  of course depends on Legal feedback, ryt?

Reply:
done